packages/headless/src/controllers/commerce/core/facets/category/headless-commerce-category-facet.test.ts

```diff
@@ -107,7 +107,7 @@ describe('CategoryFacet', () => {
   });
 
   describe('#showMoreValues', () => {
-    it('should dispatch #updateCategoryFacetNumberOfValues with initialNumberOfValues + the number of values from the request', () => {
+    it('should dispatch #updateCategoryFacetNumberOfValues with initialNumberOfValues + the number of values of the deepest selected value from the request', () => {
```

Comment:
I think this description is not quite right: in this case no value is selected, so it would be the initialNumberOfValues + the number of root values from the request.